x-pack/solutions/security/plugins/security_solution/public/common/components/header_actions/helpers.ts

```diff
@@ -46,7 +44,7 @@ export const getActionsColumnWidth = (actionButtonCount: number): number => {
   // `EuiDataGridRowCell` applies additional `padding-left` and
   // `padding-right`, which must be added to the content width to prevent the
   // content from being partially hidden due to the space occupied by padding:
-  const leftRightCellPadding = parseInt(euiThemeVars.euiDataGridCellPaddingM, 10) * 2; // parseInt ignores the trailing `px`, e.g. `6px`
+  const leftRightCellPadding = 12;
```

Comment:
this change isn't ideal either, but:>
- EUI is not currently a replacement for `euiDataGridCellPaddingM` (unless I'm missing something)
- due to the way the `getActionsColumnWidth` function is used, I'm not even sure how we would pass data to it?